### ISSUE TITLE
anvil: Newly placed windows are activated but don't receive keyboard focus

### ISSUE THREAD

**Issue by mwcampbell** [Smithay/smithay#1278]
I noticed that when a new window is placed (mapped), it's activated, but it doesn't receive keyboard focus. Is this deliberate, an oversight, or a limitation of the design?

**Comment by i509VCB** [comment]
Technically receiving keyboard focus is not required, but typically in a stacking window manager the new window will get keyboard focus.
